packages/code/src/main.rs

```diff
@@ -0,0 +1,144 @@
+use std::env;
+use std::fmt::Write;
+use std::io::{self, Read};
+
+use serde_json::{json, Value};
+use syntect::easy::HighlightLines;
+use syntect::parsing::SyntaxSet;
+use syntect::highlighting::{ThemeSet, Color};
+use syntect::html::{styled_line_to_highlighted_html, IncludeBackground};
+
+fn main() {
+    let args: Vec<String> = env::args().skip(1).collect();
+    let action = &args[0];
+    match action.as_str() {
+        "manifest" => manifest(),
+        "transform" => transform(&args[1], &args[2]),
+        other => {
+            eprintln!("Invalid action {other}")
+        }
+    }
+}
+
+fn manifest() {
+    print!("{}", serde_json::to_string(&json!(
+        {
+        "name": "Standard code package",
+        "version": "0.1",
+        "description": "This package provides syntax highlighting in [code] modules",
+        "transforms": [
+            {
+                "from": "code",
+                "to": ["html"],
+                "arguments": [
+                    {"name": "lang", "description": "The language to be highlighted"},
+                    {"name": "font_size", "default": "12", "description": "The size of the font"},
+                    {"name": "tab_size", "default": "4", "description": "The size tabs will be adjusted to"},
+                    {"name": "theme", "default": "mocha", "description": "Theme of the code section"},
+                    {"name": "bg", "default": "default", "description": "Background of the code section"},
+                ],
+            }
+        ]
+        }
+    ))
+    .unwrap());
+}
+
+fn transform(from: &String, to: &String) {
+    match from.as_str() {
+        "code" => transform_code(to),
+        other => {
+            eprintln!("Package does not support {other}");
+            return;
+        }
+    }
+}
+
+fn transform_code(to: &String) {
+    macro_rules! get_arg {
+        ($input:expr, $arg:expr) => {
+            if let Value::String(val) = &$input["arguments"][$arg] {
+                val
+            } else {
+                panic!("No theme argument was provided");
```

Comment:
I think this argument message is wrong, since you don't always get the "theme" argument using this macro, right?